gc/base/TLHAllocationSupport.cpp

```diff
@@ -419,4 +431,55 @@ MM_TLHAllocationSupport::objectAllocationNotify(MM_EnvironmentBase *env, void *h
 }
 #endif /* OMR_GC_OBJECT_ALLOCATION_NOTIFY */
 
+/**
+ * Reserve part (top) of TLH for GC if collector requires
+ */
+void
+MM_TLHAllocationSupport::reserveTLHTopForGC(MM_EnvironmentBase *env)
+{
+	uintptr_t bytesToReserveForGC = env->getExtensions()->getGlobalCollector()->reservedForGCAllocCacheSize();
```

Comment:
Body of these 2 methods could be inside #if realtime. Up to you

Reply:
If these methods get wrapped in `#if realtime` then I would prefer to put everything related (like `_reservedBytesForGC`) in if def as well. But that can get messy because references are spread out in the file, I'll leave it as is for now.